bitnami/mongodb/Chart.yaml

```
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb
   - https://mongodb.org
-version: 10.29.4
+version: 10.29.5
```

Adding new features should be a minor bump
```suggestion
version: 10.30.0
```